.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
   - "4"
   - "6"
-  - "stable"
+  - "7"
```

Comment:
Just realized `stable` is deprecated. Could we use `node` instead? I'd rather not have to update this again when we go from 7 - 8.